pkg/auth/providers/azure/azure_actions.go

```diff
@@ -70,7 +70,7 @@ func (ap *azureProvider) testAndApply(actionName string, action *types.Action, r
 	}
 
 	if azureADConfig.ApplicationSecret != "" {
-		value, err := common.ReadFromSecret(ap.secrets, azureADConfig.ApplicationSecret, "")
+		value, err := common.ReadFromSecret(ap.secrets, azureADConfig.ApplicationSecret, "applicationsecret")
```

Comment:
lets put `applicationsecret` as a constant somewhere as it's being referenced in several places. Same for other provider's secrets